src/crate/client/test_cursor.py

```diff
@@ -19,11 +19,14 @@
 # with Crate these terms will supersede the license and you may use the
 # software solely pursuant to the terms of the relevant commercial agreement.
 
-from datetime import datetime
+import datetime
+import zoneinfo
```

Comment:
`zoneinfo` is only available in Python 3.9 and newer, see https://docs.python.org/3/library/zoneinfo.html.

```
ModuleNotFoundError: No module named 'zoneinfo'
```

-- https://github.com/crate/crate-python/actions/runs/3276036472/jobs/5391606304#step:4:212

Reply:
The right solution is to probably use a backport package, see [`backports.zoneinfo`](https://pypi.org/project/backports.zoneinfo/).